app-text/pastebinit/pastebinit-1.7.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..12} )
```

Comment:
Do you have any chance to test also for python3.13?

Reply:
Just tried it with python3.13 and works as expected. Updated the ebuild.